generator/config/query/geoIntersects.yaml

```diff
@@ -11,3 +11,42 @@ arguments:
         name: geometry
         type:
             - geometry
+tests:
+    -
+        name: 'Intersects a Polygon'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/query/geoIntersects/#intersects-a-polygon'
+        pipeline:
+            -
+                $match:
+                    loc:
+                        $geoIntersects:
+                            $geometry:
```

Comment:
This requires the new `DollarObject` encoding type. https://github.com/mongodb/mongo-php-builder/pull/24/files#diff-bed00226517488779abea06643721208b8812c52dbf1c0d7f5aa01b87c5dfb69R114

Reply:
No, it's actually documented. But I think we don't need to support eh "legacy coordinates"?